app/controllers/resources_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ResourcesController < ApplicationController
+  def index
+    category_id = params[:category_id]
+    @resources = if category_id
+                   Category.find(category_id).resources
```

Comment:
For posterity: @lelandg559 reminded me that we've got a join table between the two, so what I've got doesn't work. We have to get a bit fancier to make it work.